### DISCUSSION TITLE
表单BasicForm，如果后端json有30个字段，前端只想处理10个，formSchema里配置10个，剩余20个会丢失

### DISCUSSION THREAD

**Discussion by xiatiandegaga** [vbenjs/vue-vben-admin#2633] [General]
const values = await validate();
  只会取到10个值，传到后端就有问题了，请问怎么处理合适？formSchema里用 show: false配置剩下的20个字段太冗余了。

**Comment by phphy** [comment]
我也挺疑惑的，没看到相关的demo
